packages/core/src/components/widgets/BaseInput.js

```diff
@@ -76,11 +76,13 @@ function BaseInput(props) {
     />,
     schema.examples ? (
       <datalist id={`examples_${inputProps.id}`}>
-        {schema.examples
-          .concat(schema.default ? [schema.default] : [])
-          .map(example => (
-            <option key={example} value={example} />
-          ))}
+        {[
+          ...new Set(
+            schema.examples.concat(schema.default ? [schema.default] : [])
```

Comment:
Why did you decide to include `schema.default` within the examples that show up?

Reply:
I did this because of this sentence in the [jsonschema specification](https://json-schema.org/understanding-json-schema/reference/generic.html): 
> There is no need to duplicate the default value in the examples array, since default will be treated as another example.

I agree if you think this is a bit of an odd behavior, seeing as default does not receive this special treatment when there are no examples. I do think this is most in accordance with the jsonschema spec, but I don't feel strongly about either way.

Reply:
Thanks for referring to that. I believe you linked to a summary. Looking at the spec, it seems to be really our choice as to whether we should implement it (https://datatracker.ietf.org/doc/draft-handrews-json-schema-validation/?include_text=1):

> Implementations MAY use the value(s) of "default", if present, as an
   additional example.  If "examples" is absent, "default" MAY still be
   used in this manner.

I think that what you've implemented, though, in which default is an additional example, but only shows up in "examples" if the "examples" array is defined, is a sensible way to do it.